cmd/jaeger/ingester-with-remote.yaml

```diff
@@ -0,0 +1,31 @@
+service:
+  extensions: [jaeger_storage]
+  pipelines:
+    traces:
+      receivers: [kafka]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]  # same as in the default cmd/jaeger-v2/config.yaml
+  telemetry:
+    metrics:
+      address: 0.0.0.0:8889 # to avoid port conflict with collector-with-kafka.yaml
+
+extensions:
+  jaeger_storage:
+    grpc:
+      memstore:
```

Comment:
nit: we don't know it's a `memstore` in this config, let's call it 'external-storage'